x-pack/plugins/secops/public/components/formatted_date/index.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment-timezone';
+import * as React from 'react';
+import { pure } from 'recompose';
+
+import { AppFrameworkAdapter } from '../../lib/lib';
+
+export const KibanaConfigContext = React.createContext<Partial<AppFrameworkAdapter>>({});
+
+export const PreferenceFormattedDate = pure<{ value: Date | string }>(({ value }) => (
+  <KibanaConfigContext.Consumer>
+    {(config: Partial<AppFrameworkAdapter>) =>
+      config && config.dateFormat && config.dateFormatTz
```

Comment:
if you like me and have dateFormatTz set to `Browser`, will moment understand that as a timezone? or does kibana do some magic behind the scene to give back my browser's tz

Reply:
Good point! In testing I had thought that Kibana would return the browser's tz, but it looks like it's just returning the string `Browser` instead. In `kibana_framework_adapter`, the `AppKibanaUIConfig` object does set `timezone` via the timezoneProvider, so I'll add a conditional to use that timezone if the user configured `dateFormat:tz === 'Browser'`. Thanks!